botocore/serialize.py

```diff
@@ -669,6 +845,41 @@ def _convert_header_value(self, shape, value):
             return value
 
 
+class BaseRpcV2Serializer(Serializer):
+    """Base class for RPCv2 protocols.
+
+    The only variance between the various RPCv2 protocols is the
+    way that the body is serialized.  All other aspects (headers, uri, etc.)
+    are the same and logic for serializing those aspects lives here.
+
+    Subclasses must implement the ``_serialize_body_params`` method.
+
+    """
+
+    def serialize_to_request(self, parameters, operation_model):
+        serialized = self._create_default_request()
+        service_name = operation_model.service_model.metadata['targetPrefix']
+        operation_name = operation_model.name
+        serialized['url_path'] = (
+            f'/service/{service_name}/operation/{operation_name}'
+        )
+
+        input_shape = operation_model.input_shape
+        if input_shape is not None:
+            self._serialize_payload(
+                parameters, serialized, input_shape
+            )
+
+        self._serialize_headers(serialized, operation_model)
```

Comment:
Is `X-Amz-Target` no longer relevant for RPCv2?

Reply:
We also don't appear to be implementing stubbed versions of these methods.

Reply:
From the [spec](https://smithy.io/2.0/additional-specs/protocols/smithy-rpc-v2.html):
 
>Requests for the rpcv2Cbor protocol MUST NOT contain an X-Amz-Target or X-Amzn-Target header.  An rpcv2Cbor request is malformed if it contains either of these headers. Server-side implementations MUST reject such requests for security reasons.